Orbitersdk/samples/ProjectApollo/src_sys/inertial.cpp

```diff
@@ -0,0 +1,148 @@
+/***************************************************************************
+This file is part of Project Apollo - NASSP
+Copyright 2004-2022
+
+Generic Apollo Guidance computer class which is hooked
```

Comment:
Don't forget to update this description. Also since this is a new file Copyright would be just for this year and not back to 2004. ;)